tenseal/binding.cpp

```diff
@@ -1,14 +1,13 @@
 #include <pybind11/pybind11.h>
 #include <pybind11/stl.h>
-#include <seal/seal.h>
 
 #include <memory>
 #include <vector>
 
-#include "tenseal/tensealcontext.h"
-#include "tenseal/tensors/bfvvector.h"
-#include "tenseal/tensors/ckksvector.h"
-#include "tenseal/utils/utils.h"
+#include "seal/seal.h"
```

Comment:
is this okay to include it like this? shouldn't it be considered like an installed library?

Reply:
If I include it with <seal/seal.h>, it will search the system headers too.
If I include it with "seal/seal.h", it will search just the linked headers in the build process.

It can lead no weird bugs due to system artifacts, we should use our headers.